### PR TITLE
Fix default failure reason

### PR DESCRIPTION
**Pull request by cnnorris**

### CODE REVIEW

File: portability-spi-cloud/src/main/java/org/datatransferproject/spi/cloud/types/PortabilityJob.java [hunk 1]
```diff
@@ -259,6 +263,7 @@ public boolean equals(Object o) {
     @JsonProperty("lastUpdateTimestamp")
     public abstract Builder setLastUpdateTimestamp(LocalDateTime lastUpdateTimestamp);
 
+    @JsonInclude(JsonInclude.Include.NON_NULL)
```
Comment (seehamrun) [inline portability-spi-cloud/src/main/java/org/datatransferproject/spi/cloud/types/PortabilityJob.java]: what does this NON_NULL include do?

Comment (cnnorris) [reply portability-spi-cloud/src/main/java/org/datatransferproject/spi/cloud/types/PortabilityJob.java]: it only gets included if it's specified: https://fasterxml.github.io/jackson-annotations/javadoc/2.0.0/com/fasterxml/jackson/annotation/JsonInclude.Include.html

Comment (cnnorris) [reply portability-spi-cloud/src/main/java/org/datatransferproject/spi/cloud/types/PortabilityJob.java]: I copied it from l256

Comment (seehamrun) [reply portability-spi-cloud/src/main/java/org/datatransferproject/spi/cloud/types/PortabilityJob.java]: nifty!

**Pull request merged**